src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -24,6 +24,8 @@
 import ch.njol.skript.aliases.ItemData;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.util.Color;
+import org.skriptlang.skript.lang.comparator.Comparator;
```

Comment:
Unsure if the suggestion is right I can't read git for the life of me
```suggestion
import org.skriptlang.skript.lang.comparator.Comparator;
```

Reply:
Thank you for the review however, we have decided to keep this band aid fix to be able to use this PR until we make a proper fix for that.